src/inputs/getViteConfigPath.ts

```diff
@@ -10,6 +10,12 @@ const testFilePath = async (workingDirectory: string, filePath: string) => {
 };
 
 const defaultPaths = [
+  "vitest.config.ts",
+  "vitest.config.mts",
+  "vitest.config.cts",
+  "vitest.config.js",
+  "vitest.config.mjs",
+  "vitest.config.cjs",
```

Comment:
I guess this was supposed to be `vitest.workspace.x`?

Could you also please add those at the end of the array rather than the top?